pmg/models/users.py

```diff
@@ -146,6 +149,18 @@ def gets_alerts_for(self, committee):
             committee = Committee.query.get(committee)
         return committee in self.committee_alerts
 
+    def follow_committee(self, committee):
+        from ..models.resources import Committee
+        if not isinstance(committee, Committee):
+            committee = Committee.query.get(committee)
+        self.following.append(committee)
+
+    def unfollow_committee(self, committee):
+        from ..models.resources import Committee
+        if not isinstance(committee, Committee):
+            committee = Committee.query.get(committee)
+        self.following.remove(committee)
```

Comment:
The idea is that if you aren't following a cte, then you won't get alerts either. So here we should also remove the cte from the `committee_alerts` list.